tools/RAiDER/statsPlot.py

```diff
@@ -48,6 +51,8 @@ def create_parser():
                           help='Specified input unit. Input will be converted into m if not already in m.')
     userinps.add_argument('-w', '--workdir', dest='workdir', default='./',
                           help='Specify directory to deposit all outputs. Default is local directory where script is launched.')
+    userinps.add_argument('--cpus', dest='numCPUs', type=parse_cpus, default=8,
+                          help='Specify number of cpus to be used for multiprocessing. May specify "all" at your own discretion.')
```

Comment:
We could split the `create_parser` function into multiple functions so that we only need to define shared arguments in one place. For example:

```python3
def add_common_arguments(parser):
    parser.add_argument('--verbose, ...)
    parser.add_argument('--cpus', ...)
    ...
```

I'm not saying to necessarily do it in this PR, but it's something to keep in mind for the future, so we can avoid copy/pasting the same strings around.

Reply:
Yeah, probably better to address in a separate PR. Will raise an issue ticket.